.github/workflows/build-docker.yaml

```diff
@@ -31,6 +40,15 @@ jobs:
             - name: Set up Docker Buildx
               uses: docker/setup-buildx-action@v3
 
+            - name: Determine Image Tag
+              if: ${{ github.ref_type == 'branch' }}
+              run: |
+                  echo \"IMAGE_TAG=${{env.NEXT_RELEASE_VERSION}}-pre-release-$(git rev-parse --short ${{ github.sha }})\" >> $GITHUB_ENV
```

Comment:
Will NEXT_RELEASE_VERSION be available? Since the semantic release version runs a a different job.